build/Dockerfile

```diff
@@ -14,36 +14,39 @@ RUN echo "4a7fa60f323ee1416a4b1425aefc37ea359e9d64df19c326a58953a97ad41ea5  go.t
 RUN tar -C /usr/local -xzf go.tar.gz
 ENV PATH=${PATH}:/usr/local/go/bin
```

Comment:
```suggestion
ENV GOBIN=/usr/local/go/bin
ENV PATH=${PATH}:${GOBIN}
```

Reply:
@PratikMahajan seems this was overlooked. You'll also need to make the changes you just pushed to the CI Dockerfile